pkg/kubelet/status/testing/mock_pod_status_provider.go

```diff
@@ -90,32 +90,32 @@ func (m *MockPodDeletionSafetyProvider) EXPECT() *MockPodDeletionSafetyProviderM
 	return m.recorder
 }
 
-// PodCouldHaveRunningContainers mocks base method.
-func (m *MockPodDeletionSafetyProvider) PodCouldHaveRunningContainers(pod *v1.Pod) bool {
+// PodResourcesAreReclaimed mocks base method.
+func (m *MockPodDeletionSafetyProvider) PodResourcesAreReclaimed(pod *v1.Pod, status v1.PodStatus) bool {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "PodCouldHaveRunningContainers", pod)
+	ret := m.ctrl.Call(m, "PodResourcesAreReclaimed", pod, status)
 	ret0, _ := ret[0].(bool)
 	return ret0
 }
 
-// PodCouldHaveRunningContainers indicates an expected call of PodCouldHaveRunningContainers.
-func (mr *MockPodDeletionSafetyProviderMockRecorder) PodCouldHaveRunningContainers(pod interface{}) *gomock.Call {
+// PodResourcesAreReclaimed indicates an expected call of PodResourcesAreReclaimed.
+func (mr *MockPodDeletionSafetyProviderMockRecorder) PodResourcesAreReclaimed(pod, status interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "PodCouldHaveRunningContainers", reflect.TypeOf((*MockPodDeletionSafetyProvider)(nil).PodCouldHaveRunningContainers), pod)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "PodResourcesAreReclaimed", reflect.TypeOf((*MockPodDeletionSafetyProvider)(nil).PodResourcesAreReclaimed), pod, status)
 }
 
-// PodResourcesAreReclaimed mocks base method.
-func (m *MockPodDeletionSafetyProvider) PodResourcesAreReclaimed(pod *v1.Pod, status v1.PodStatus) bool {
+// ShouldDelayTransitioningPodToTerminal mocks base method.
+func (m *MockPodDeletionSafetyProvider) ShouldDelayTransitioningPodToTerminal(pod *v1.Pod) bool {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "PodResourcesAreReclaimed", pod, status)
+	ret := m.ctrl.Call(m, "ShouldDelayTransitioningPodToTerminal", pod)
 	ret0, _ := ret[0].(bool)
 	return ret0
 }
 
-// PodResourcesAreReclaimed indicates an expected call of PodResourcesAreReclaimed.
-func (mr *MockPodDeletionSafetyProviderMockRecorder) PodResourcesAreReclaimed(pod, status interface{}) *gomock.Call {
+// ShouldDelayTransitioningPodToTerminal indicates an expected call of ShouldDelayTransitioningPodToTerminal.
+func (mr *MockPodDeletionSafetyProviderMockRecorder) ShouldDelayTransitioningPodToTerminal(pod interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "PodResourcesAreReclaimed", reflect.TypeOf((*MockPodDeletionSafetyProvider)(nil).PodResourcesAreReclaimed), pod, status)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "ShouldDelayTransitioningPodToTerminal", reflect.TypeOf((*MockPodDeletionSafetyProvider)(nil).ShouldDelayTransitioningPodToTerminal), pod)
```

Comment:
Can you rearrange these to reduce the diff?

Reply:
I can't. It was generated by the `hacks/update-mocks.sh`

Reply:
ack